packages/big-design/src/components/MultiSelect/spec.tsx

```diff
@@ -1085,6 +1085,41 @@ test('renders localized labels', async () => {
   expect(label).toHaveStyleRule('content', "' (opcional)'", { modifier: '::after' });
 });
 
+test('selects all triggers onOptionsChange with available options', async () => {
```

Comment:
Can you add a test for indeterminate and unselect all?

Reply:
Yes, added two more tests to cover these cases